packages/utils/src/cloudwatch/inbound-patient-lambda.ts

```diff
@@ -0,0 +1,213 @@
+/**
+ * Script to query CloudWatch logs for occurrences of "Match: true" over the past two weeks.
+ *
+ * This script performs the following actions:
+ * - Iterates through each day in the past two weeks
+ * - Queries CloudWatch logs for the specified log group and each day's time range
+ * - Counts the number of times "Match: true" appears in the logs for each day
+ * - Outputs the total count per day and the cumulative total
+ */
+
+import { queryLogs } from "./query";
+import { initDbPool } from "@metriport/core/util/sequelize";
+import { QueryTypes } from "sequelize";
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+import * as fs from "fs";
+import * as path from "path";
+
+/**
+ * Configuration
+ */
+const LOG_GROUP_NAME = "/aws/lambda/IHEInboundPatientDiscoveryV2Lambda";
+const FILTER_PATTERN = '"Match: true"';
+const DAYS_TO_QUERY = 15;
+
+const cxIdMap: Map<string, number> = new Map();
+const recentPatientCountMap: Map<string, number> = new Map();
+const patientIdSet: Set<string> = new Set();
+
+const CX_IDS: string[] = [];
```

Comment:
This is configuration, so let's concentrate those at the respective section? Then separate what's regular maps, consts and other objects/data structures used by the script but not part of the setup.